.github/workflows/test.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby-version: ['3.0', '3.1', '3.2', '3.3']
```

Comment:
Hmmm.. Maybe instead of removing ruby 3.0, the specific grape version can be set instead. As a separate PR ofc.

Reply:
For the reference: https://github.com/ruby-grape/grape-swagger/blob/master/.github/workflows/ci.yml#L28

Reply:
That's a good idea, will merge this one, feel free to pick it up?

Reply:
Sure! Will do this.

Reply:
As promised: #76